src/test/java/org/jpeek/calculus/java/Lcom4Test.java

```diff
@@ -65,18 +64,19 @@ public void createsXmlCalculusWithXpaths() throws IOException {
         ).affirm();
     }
 
-    @Test
+    @ParameterizedTest
     @Disabled
-    public void methodMethodCalls() throws Exception {
+    @CsvFileSource(resources = "/org/jpeek/calculus/java/lcom4-params.csv")
+    public void methodMethodCalls(final String file, final String value) throws Exception {
```

Comment:
@HDouss I know that is not your change, but please change the name of the method to a meaningful one